urnik/templates/navigation.html

```diff
@@ -1,2 +1,2 @@
-<li id="domov"><a href="{% url 'zacetna_stran' %}"><i class="material-icons left">home</i>Urnik FMF</a></li>
-<li id="rezervacije"><a href="{% url 'rezervacije' %}"><i class="material-icons left">border_color</i>Rezervacije</a></li>
```

Comment:
Te dva id-ja se nikjer ne uporabljata, napaka ej pa bila da je navigacija includana dvakrat, in ne smemo imeti podvojenih idjev.